meetups/Meetup_Best_Practices.md

```diff
@@ -0,0 +1,52 @@
+## Meetups Best Practices
+
+This page allows you to start a new meetup group based on the experience of existing ones. <br>
+Here is a list of suggestions and tips to get you started:
+
+* **Create the meetup**
+
+The **description** of the group is left to the discretion of the organizers. <br>
+A brief description of the role of the CNCF and the Cloud Native approach could be added. <br>
+Here is an example based on the [FAQ](https://www.cncf.io/about/faq/)
+
+```
+Cloud native computing uses an open source software stack to be:
+1 - Containerized. Each part (applications, processes, etc) is packaged in its own container. This facilitates reproducibility, transparency, and resource isolation.
+2 - Dynamically orchestrated. Containers are actively scheduled and managed to optimize resource utilization.
+3 - Microservices oriented. Applications are segmented into microservices. This significantly increases the overall agility and maintainability of applications.
+```
+
+In order for your meetup to be the most visible you should add a relevant list of **tags**. <br>
+(e.g. Cloud Computing · DevOps · Native Cloud Computing · Docking · Microservices · Kubernetes · Cloud Native · Delivery ...)
+
+The most important thing is to keep a **active group**, with frequent events (ideally once a month). <br>
+Animating a group alone can be difficult to manage, if it is possible you would be several co-organizers.
+
+
+* **Find Speakers**
+
+You should provided some kind of **form** in order to allow people to submit their talk proposal as follows :
+
+https://docs.google.com/forms/d/1V2Y03YMOrIor0M796_WMbYx-fdsn80ngaT-PIum8gUU/edit <br>
+_**Note**_: Do a copy ! Please do not edit the form in place.
+
```

Comment:
@idvoretskyi i've applied the changes according to your suggestions

Reply:
LGTM